Modules/_elementtree.c

```diff
@@ -69,13 +69,6 @@ static void _clear_joined_ptr(PyObject **p)
     }
 }
 
-/* Types defined by this extension */
-static PyTypeObject *Element_Type;
-static PyTypeObject *ElementIter_Type;
-static PyTypeObject *TreeBuilder_Type;
-static PyTypeObject *XMLParser_Type;
```

Comment:
Probably, these variables should be deleted from the c-analyzer list as well:

https://github.com/python/cpython/blob/78d1676336491ffc0279da0a9f6d7e91252be783/Tools/c-analyzer/cpython/globals-to-fix.tsv#L428-L431

Reply:
Very good catch; thanks.